lib/semian/atomic_enum.rb

```diff
@@ -0,0 +1,39 @@
+require 'forwardable'
+
+module Semian
+  class AtomicEnum #:nodoc:
+    extend Forwardable
+
+    def_delegators :@integer, :execute_atomically, :transaction, :shared?, :destroy
+    private :shared?
+
+    def initialize(symbol_list, **_options)
```

Comment:
I don't understand, why does this have an `options` parameter? You are not using those values anywhere, are you?